Outerloop failures GC/LargeMemory missing projects

```
/Users/runner/runners/2.165.0/work/1/s/.dotnet/sdk/5.0.100-preview.1.20112.7/Microsoft.Common.CurrentVersion.targets(1850,5):
 warning : The referenced project 'largeobject.csproj' does not exist.
 [/Users/runner/runners/2.165.0/work/1/s/src/coreclr/tests/src/GC/LargeMemory/Allocation/largeexceptiontest.csproj]

##[warning].dotnet/sdk/5.0.100-preview.1.20112.7/Microsoft.Common.CurrentVersion.targets(1850,5):
 warning : The referenced project 'largeobject.csproj' does not exist.
/Users/runner/runners/2.165.0/work/1/s/.dotnet/sdk/5.0.100-preview.1.20112.7/Microsoft.Common.CurrentVersion.targets(1850,5):
 warning : The referenced project 'memcheck.csproj' does not exist. [/Users/runner/runners/2.165.0/work/1/s/src/coreclr/tests/src/GC/LargeMemory/Allocation/largeexceptiontest.csproj]

##[warning].dotnet/sdk/5.0.100-preview.1.20112.7/Microsoft.Common.CurrentVersion.targets(1850,5):
 warning : The referenced project 'memcheck.csproj' does not exist.
  largeobject -> /Users/runner/runners/2.165.0/work/1/s/artifacts/tests/coreclr/OSX.x64.Checked/GC/LargeMemory/largeobject/largeobject.dll
largeexceptiontest.cs(12,5): error CS0246:
 The type or namespace name 'LargeObject' could not be found (are you missing a using directive or an assembly reference?) [/Users/runner/runners/2.165.0/work/1/s/src/coreclr/tests/src/GC/LargeMemory/Allocation/largeexceptiontest.csproj]

##[error]largeexceptiontest.cs(12,5):
 error CS0246: The type or namespace name 'LargeObject' could not be found (are you missing a using directive or an assembly reference?)

```

From merges in master e.g. https://dev.azure.com/dnceng/public/_build/results?buildId=540488&view=logs&jobId=721a035e-086a-5f66-d7c2-78d896701c82&j=721a035e-086a-5f66-d7c2-78d896701c82&t=d90f58c6-db3f-521d-96d8-db7a4001b9f4

--- a/src/coreclr/tests/src/GC/LargeMemory/Allocation/largeexceptiontest.csproj
+++ b/src/coreclr/tests/src/GC/LargeMemory/Allocation/largeexceptiontest.csproj
@@ -9,7 +9,7 @@
     <Compile Include="largeexceptiontest.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="largeobject.csproj" />
-    <ProjectReference Include="memcheck.csproj" />
+    <ProjectReference Include="../largeobject.csproj" />
+    <ProjectReference Include="../memcheck.csproj" />
   </ItemGroup>
 </Project>

